### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by masraniJay**
Added landing page.

### CODE REVIEW

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
It looks good

**Review by dusariudaykumar — APPROVED** [review]
looks good

File: index.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+</head>
+
+<body>
+    <header class="navigation">
+        <a class="navigation-logo">Feather-store</a>
+        <input class="navigation-search" placeholder="Search">
+        <div>
+            <a href="" class="navigation-item">Login</a>
+            <a href="" class="navigation-item">Whishlist</a>
+            <a href="" class="navigation-item">Cart</a>
```
Comment (shubhmehra) [inline index.html:20]: why empty hrefs?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                <img class="card-image" src="assets/kidswear.jpg">
+                <div class="card-title">
+                    <h3>Summer Collection</h3>
+                    <br>
+                    <small>Check out out best summer collection to stay warm in style this season!</small>
+                </div>
+            </div>
+
+            <div class="card card-new-arrival">
+                <img class="card-image" src="assets/kidswear.jpg">
```
Comment (shubhmehra) [inline index.html:95]: please use alt tags for better accessibility

File: index.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+                </div>
+            </div>
+
+        </div>
+
+    </div>
+
+</body>
+
+</html>
```
Comment (shubhmehra) [inline index.html:145]: code formatting need to

**Review by shubhmehra — APPROVED** [review]
please use prettier for code formatting
otherwise looks good

**Pull request merged**